controllers/manager.go

```diff
@@ -120,9 +121,11 @@ func (o *ManagerOptions) runManager() {
 
 	namespace := os.Getenv("POD_NAMESPACE")
 
+	logger := slog.New(slog.NewTextHandler(os.Stdout, nil))
```

Comment:
Need JSON handler for production and Text handler for development. Use envvar or flags to switch between those.

Reply:
added flag (--dev) to switch between json and text handlers